packages/dnb-design-system-portal/src/docs/uilib/components/number-format/Examples.js

```diff
@@ -66,6 +66,37 @@ export const NumberCurrency = () => (
   </Style>
 )
 
+export const NumberCompact = () => (
+  <Style>
+    <ComponentBox data-visual-test="number-format-compact">
+      {() => /* jsx */ `
+<P>
+  <NumberFormat compact>1234</NumberFormat>
+  <NumberFormat
+    compact="long"
+    decimals={1}
+    value={12345}
+  />
+  <NumberFormat
+    compact
+    currency
+    decimals={2}
+    value={-1723967.38}
+  />
+  <NumberFormat
+    compact
```

Comment:
```suggestion
    compact="short"
```

Just a suggestion to see an example of "short" version as well.

Reply:
ahh, det short version is the default – will make it more clear in the docs 👍